Packs/JoeSecurity/Integrations/JoeSecurityV2/JoeSecurityV2.py

```diff
@@ -22,8 +22,7 @@ def __init__(self, apikey: str = '', on_premise: bool = True, base_url: str = ''
         self.reliability = reliability
         self.create_relationships = create_relationships
         self.on_premise = on_premise
-        if proxy:
-            proxies = handle_proxy()
+        proxies = handle_proxy()
```

Comment:
Add debug log here to print in case proxy is enabled print handling proxy
in case proxy is false print removing proxy from env vars

Reply:
it's done inside CSP not specific for Joe security

Reply:
Log calling handle_proxy() in the integration